tools/benchmark-core/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
Curious, why not put it in yew's benches instead of creating a new crate?

Reply:
so what do you prefer? should I move the bench in the crate `yew`? or keep with a separate crate for consistency sake? both are fine imo i don't really care

Reply:
I'm fine either way. It doesn't really make that much of a difference and moving it now is just extra work